utils/lib/openqa_api.py

```diff
@@ -230,7 +233,14 @@ def get_results(self):
                         if not TestFailure.exists_in_db(failure):
                             local_session.add(failure)
                             local_session.flush()
-                        failure_list.append(failure)
+                        failure_per_group[failure.name].append(failure)
+                    elif failure.name == "system_tests":
+                        delayed_failure_per_group[failure.name].append(failure)
+
+            if not failure_per_group[test_group['name']]:
```

Comment:
Does this work at all? If upload failed, you won't see those other groups at all...

Reply:
Did it? I don't see audio failures listed for example (https://openqa.qubes-os.org/tests/150378)